roles/development/files/docker-elasticsearch.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
```

Comment:
Hi @wakawakaxin 

could you move the docker related tasks to a separated role named "docker". We will use the docker images in all environments